src/app/public/modules/split-view/split-view-workspace.component.html

```diff
@@ -0,0 +1,13 @@
+<div
+  class="sky-split-view-workspace"
+  role="region"
+  [attr.aria-label]="ariaLabel"
+>
+  <sky-split-view-workspace-header *ngIf="isMobile"
```

Comment:
The spacing here seems odd. I think we would be good to either put this all on one line or the full open tag on one and the close on the next.

Reply:
Done.